### PR TITLE
Add system test for HSTS and X-Content-Type vulnerabilities detection

### PR DESCRIPTION
**Pull request by uurien**
## Description

Add test to check the detection of two missing headers vulnerabilities. The endpoints are implemented only in nodejs application.

## Reviewer checklist

* [ ] If this PR modifies anything else than strictly the default scenario, then remove the `run-default-scenario` label
* [x] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

> **_NOTE:_**  By default in PR only default scenario tests will be launched. Remove the `run-default-scenario` label to run all scenarios ([more info](https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2866381467/CI+Workflow+Github+Actions))

Once your PR is reviewed, you can merge it! :heart:

### CODE REVIEW

File: tests/appsec/iast/sink/test_hsts_missing_header.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+class TestHstsMissingHeader:
+    """Test Hsts missing header detection."""
+
+    sink_fixture = SinkFixture(
+        vulnerability_type="HSTS_HEADER_MISSING",
+        http_method="GET",
+        insecure_endpoint="/iast/hsts-missing-header/test_insecure",
+        secure_endpoint="/iast/hsts-missing-header/test_secure",
+        data={},
+        request_kwargs={"headers": {"X-Forwarded-Proto": "https"}},
```
[the remark this answers is not in the document]
Comment (uurien) [reply tests/appsec/iast/sink/test_hsts_missing_header.py:24]: I did it at this way just to do it in the same way as SourceFixture.

**Review by DDJavierSantos — APPROVED** [review]

**Comment by cbeauchesne** [comment]
@uurien is this PR still relevant ?

**Comment by uurien** [comment]
I completely forgot about this one 🤦 
Now that @DDJavierSantos made the PR for Java, I'll create new PR after his merge and close this one.

**Comment by uurien** [comment]
Closing this PR in favour of https://github.com/DataDog/system-tests/pull/1636

**Pull request closed without merging**